### PR TITLE
feat: update avatar

### LINKED ISSUE #285: user avatar on gamedao
## User Story
As user I want to see an avatar which looks always the same  so I dont get confused

## Acceptance Criteria

1. instead of using gravatar use the avatar in the wallet view on the account page and membership/voters/contributors lists.

## Additional Information

right avatar: 
image
wrong avatars:
image
image

## How to test

1. a
2. b
3. c
4. d

### PR DESCRIPTION
**Pull request by hadi-ibra**
closes #285
-----------------

Done:

- updated avatar on identity page
- updated avatar on organization members page

### CODE REVIEW

File: pkg/app/src/components/Account/modules/identitySection.tsx [hunk 1]
```diff
@@ -22,6 +18,7 @@ export function IdentitySection() {
 		push(`/account/${AccountTabs.IDENTITY}`)
 	}, [push])
 	const { identity } = useIdentityByAddress(getAddressFromAccountState(accountState))
+	const avatarHash = md5(getAddressFromAccountState(accountState))
```
Comment (DarkNebula0) [inline pkg/app/src/components/Account/modules/identitySection.tsx:21]: use useMemo

File: pkg/app/src/components/TabPanels/Organization/organizationMembers.tsx [hunk 2]
```diff
@@ -36,14 +36,18 @@ export function OrganizationMembersTable({ organizationState }: ComponentProps)
...
-									height: 42,
-									width: 42,
-								}}
-							>
-								{getInitials(params.row.name)}
-							</Avatar>
+								sx={{ height: 42, width: 42 }}
+								src={
+									params?.row?.email
+										? `https://avatars.dicebear.com/api/pixel-art-neutral/${md5(
```
Comment (DarkNebula0) [inline pkg/app/src/components/TabPanels/Organization/organizationMembers.tsx:42]: Simplify

	src={`https://avatars.dicebear.com/api/pixel-art-neutral/${md5(
												params?.row?.email || params?.row?.address,
										  )}.svg`
		    }

**Review by DarkNebula0 — CHANGES_REQUESTED** [review]

File: pkg/app/src/components/Account/modules/identitySection.tsx (hunk 1 above)
Comment (hadi-ibra) [reply pkg/app/src/components/Account/modules/identitySection.tsx:21]: notes

File: pkg/app/src/components/TabPanels/Organization/organizationMembers.tsx (hunk 2 above)
Comment (hadi-ibra) [reply pkg/app/src/components/TabPanels/Organization/organizationMembers.tsx:42]: yes noted

Comment (hadi-ibra) [reply pkg/app/src/components/TabPanels/Organization/organizationMembers.tsx]: yes noted

**Review by DarkNebula0 — APPROVED** [review]

**Pull request merged**